### ISSUE TITLE
Premiumize.me

### ISSUE THREAD

**Issue by gwilly7** [jsergio123/script.module.resolveurl#8]
I have been having problems with premiumize.me skipping over links since the 5.0.07 update where is says premiumize was updated. I really appreciate your hard work. Thank you.

**Comment by jsergio123** [comment]
I need a debug log of the issues.

**Comment by gwilly7** [comment]
[01_KODI.log](https://github.com/jsergio123/script.module.resolveurl/files/1801160/01_KODI.log)

Here you go. Thank you for you help. Played 1st real-debrid link 1st  and played fine then went to 1st premiumize link and skipped through them.

**Comment by jsergio123** [comment]
I do not see any errors in your log relating to ResolveURL. Are you sure this log was taken when trying to play a Premiumize source?

**Comment by gwilly7** [comment]
[kodi.log](https://github.com/jsergio123/script.module.resolveurl/files/1804831/kodi.log)
 Here is another log, this time from a windows machine. the other log was from a libreelec machine. kodi logfile uploader was failing due to size over 1mb. I just pulled this out of userdata folder. It skips over about 15 or so links before playing one. Normally with both real-debrid and premiumize pretty much anylink I pick plays.

**Comment by jsergio123** [comment]
As far as I can tell the plugin is functioning normally. Your issue may lie with the addon you are using but I will dig deeper and do some internal testing to make sure this is the case. As for the update to the PM plugin in 5.0.07's update, it was only minimal and did not alter functionality. You should try to manually rollback ResolverURL prior to .07 and see if the problem still persists. Get it here https://github.com/jsergio123/zips/raw/master/script.module.resolveurl/script.module.resolveurl-5.0.06b.zip

**Comment by gwilly7** [comment]
So I get the same problem with the old version of resolveurl.  I am so sorry, must just be a coincidence that the problems started about the same time as the update before last.  I did try another addon and got the same result.  Same links play fine with real-debrid though.  I thought is was maybe something on my end until I saw someone of reddit with the same issue.  I will check with premiumize support.   It is happening on all devices, windows, libreelec and firestick. I also tested at work to see if it might be isp related.
